src/components/LoginPage.js

```diff
@@ -230,7 +234,7 @@ const LoginPage = ({ theme }) => {
 
   return (
     <Form
-      initialValues={{ base_url: base_url }}
+      initialValues={{ base_url: override_server ? override_server : base_url }}
```

Comment:
```suggestion
      initialValues={{ base_url: override_server || base_url }}
```
`||` returns the "truthy" value, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR